presto-docs/src/main/sphinx/installation/deploy-docker.rst

```diff
@@ -0,0 +1,116 @@
+=================================
+Deploy Presto From a Docker Image
+=================================
+
+These steps were developed and tested on Mac OS X, on both Intel and Apple Silicon chips. 
+
+Following these steps, you will:
+
+- install the command line tools for brew, docker, and `colima <https://github.com/abiosoft/colima>`_ tools
+- verify your Docker setup
+- pull the Docker image of the Presto server and the Presto CLI
+- start your local Presto server and Presto CLI
+- query your local Presto server using the Presto CLI
+- query a remote Presto server using the Presto CLI
+
+Installing brew, docker, docker-compose, and colima
```

Comment:
I don't see where the docker-compose is used in this doc. am I right? But I guess it's nice to install it for other tutorials :-)

Reply:
Good catch! You're correct it's not used here directly.